tests/test_service_routes.py

```diff
@@ -166,6 +167,15 @@ def test_create_shopcart_with_pure_json(self):
         #    new_shopcart["update_time"], "Update time not set"
         #)
 
+    def test_get_pet_list(self):
+        """ Get a list of Pets """
```

Comment:
You missed changing from pet to shopcart here

Reply:
changed. that is embarrassing...